### PR TITLE
New header src ip

### PR DESCRIPTION
**Pull request by LivingInSyn**
Pull request for issue:
https://github.com/OtisElevatorCompany/udp_rx/issues/8

Supports a versioned header, IPv4, IPv6 and specifying a 'sender' IP address for adapters that have multiple IP addresses assigned to a single adapter.

### CODE REVIEW

File: header_format.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# udp_rx Header Format
+
+## Description
+udp_rx supports IPv4 and IPv6 and has a variable length header to accomidate the different lengths in addressing. udp_rx also supports multiple IP addresses assigned to a single adapter. To support this, a 'sender' IP address can be added to the header to force udp_rx to use that IP address as the sending IP.
```
Comment (kathleenm678) [inline header_format.md]: "accommodate"

File: python_helpers/rx_udp.py [hunk 2]
```diff
@@ -31,5 +32,17 @@
...
+        data = data[6:]
+        ipstring = ''
+        for octet in ip:
+            ipstring = ipstring + str(octet) + "."
+        port = (port[0] << 8) + port[1]
+        print("IP: ", ipstring)
+        print("port: ", port)
+        print("data: ", data)
+    else:
+        print("received message: {}\tfrom: {}".format(data, addr))
```
Comment (kathleenm678) [inline python_helpers/rx_udp.py]: There's a warning that says there's no newline at the end of this file. Does that need to be fixed?

File: python_helpers/tx_udp_sender.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# Copyright 2018 Otis Elevator Company. All rights reserved.
+# Use of this source code is govered by the MIT license which
+# can be found in the LICENSE file.
+
+# Author: Jeremy Mill: jeremy.mill@otis.com
+
+# Otis udp_rx software has been designed to utilize information
+# security technology described Part 774 of the EAR Category 5 Part 2
```
Comment (kathleenm678) [inline python_helpers/tx_udp_sender.py:8]: "described _in_" This issue might be in all of the files. If so don't worry about it.

**Review by kathleenm678 — APPROVED** [review]

File: python_helpers/rx_udp.py (hunk 2 above)
Comment (LivingInSyn) [reply python_helpers/rx_udp.py]: not a deal breaker, but I fixed it regardless

File: python_helpers/tx_udp_sender.py (hunk 3 above)
Comment (LivingInSyn) [reply python_helpers/tx_udp_sender.py:8]: it is, we got that from lawyers, we can't change it

**Pull request merged**